ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.2.8f1
-m_EditorVersionWithRevision: 2019.2.8f1 (ff5b465c8d13)
+m_EditorVersion: 2019.3.0f5
```

Comment:
Can we have this updated to the last 2019.3 version?

Reply:
I don't have it. I will leave this to future work.